docgen/src/guide/Handling_cache.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Handling cache
+mainTitle: Guides
+layout: main.pug
+category: guide
+navWeight: 15
+---
+
+
+By default, Algolia caches the search results of the queries, storing them locally in the cache. If the user ends up entering a search (or part of it) that has already been entered previously, the results will be retrieved from the cache, instead of requesting them from Algolia, making the application much faster.
+
+While it is a very convenient feature, in some cases, it is useful to have the ability to clear the cache and make a new request to Algolia. For instance, when changes are made on some records on your index, you might want the frontend side of your application to be updated to reflect that change (in order to avoid displaying stale results retrieved from the cache).
+
+To do so, there is a prop on the `<InstantSearch>` component called `refresh` that, when set to true, clears the cache and triggers a new search.
+
+There are two different approaches to handle the cache:
```

Comment:
Proposal:

> There are two different use cases where you would want to discard the cache:
> - your application data is being updated by another process that you don't manager, for example a CRON job updates users inside Algolia. For this you might want to periodically refresh the cache of your app
> - you application data is being updated by the users of your app directly (for example, in a dashboard). In this use case you would want to refresh the cache base on some application state like the last modification of the user

wdyt?

I put this to better explains the scenarios.

Reply:
> that you don’t manage